project.clj

```diff
@@ -36,6 +36,7 @@
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [puppetlabs/i18n ~i18n-version]
                  [cheshire "5.6.1"]
+                 [digest "1.4.4"]
```

Comment:
Adding this dependency to hash bytes, other functions we have (that use this same library) hash strings. We have a difference in version here and in kitchensink, we should probably switch to 1.4.3 so we don't conflict